security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
+//OpenShift Compliance Operator Release Notes
+[id="compliance-operator-release-notes"]
+= Compliance Operator release notes
+:context: compliance-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[id="compliance-operator-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
+
+[id="compliance-operator-release-notes-0-1-44"]
+== Release Notes for Compliance Operator 0.1.44
+Issued: 2021-
+
+The Compliance Operator 0.1.44 is now available with enhancements and bug fixes.
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+
+* To control whether scan scheduling is strict or not strict - a new `strictNode` option is available for `ComplianceScan`, `ComplianceSuite` and `ScanSetting` scans. This option defaults to true (strict), meaning that the operator will error out if a scan pod can't be scheduled. Switching the option to true makes the scan more permissive an go forward. Useful for clouds with ephemeral nodes.
++
+* In this release, the `nodeSelector` and `tolerations` in the `ScanSettings` object are now configurable  This enables deployers to configure where the Result Server will run, and thus what node will host the persistent volume that will contain the raw results. This allows us to schedule a pod that makes use of a persistent volume on the master nodes.
```

Comment:
Feature: The ScanSetting object now allows the user to configure the nodeSelector and tolerations used to place the resultserver pod (a pod where the raw ARF results are uploaded to, the pod mounts a PV storage volume).

Reason: Previously, the nodeSelector and the tolerations defaulted to selecting master nodes and tolerating the `node-role.kubernetes.io/master` taint. This doesn't work in all environments, in some, master nodes are not permitted to mount any PVs. Those environments must have a way to select other nodes and tolerate different taints.

Result: Users are able to customize which nodes are used to schedule the ResultServer workload using the nodeSelector and tolerations attributes of the ScanSetting object

Reply:
In this case, can we substitute control plane host for master node? e.g. Previously, the nodeSelector and the tolerations defaulted to selecting control plane host and tolerating the node-role.kubernetes.io/master taint

Reply:
I see the reasoning, but the fact is that the name of the role as well as the labels are called what they are now.
In the example you wrote, the second sentence:

This also allows you to specify the resultserver pod that mounts a PV storage volume used to store the raw Asset Reporting Format (ARF) results.

should say something like:
This also allows you to specify the nodes where the resultserver pod that mounts a PV storage volume used to store the raw Asset Reporting Format (ARF) results would be scheduled.